commcare_connect/templates/opportunity/user_learn_progress.html

```diff
@@ -4,9 +4,24 @@
 {% load duration_minutes %}
 
 {% block content %}
-  <div class="container">
-    <h2>Learn Progress for {{ object.user.name }}</h2>
-    <table class="table table-bordered table-condensed">
+  <nav class="mt-2 small" aria-label="breadcrumb">
+    <ol class="breadcrumb bg-white fw-bold">
+      <li class="breadcrumb-item">
+        <a href="{% url 'opportunity:list' org_slug=request.org.slug %}">
+          Opportunities
+        </a>
+      </li>
+      <li class="breadcrumb-item">
+        <a href="{% url 'opportunity:detail' org_slug=request.org.slug pk=object.pk %}">
+          The Opportunity
```

Comment:
This can be replaced with the suggestion to render the opportunity name.
```suggestion
          {{ object.opportunity.name }}
```